### ISSUE TITLE
List of everything that is required for the big quarterly/six monthly/yearly reports

### ISSUE THREAD

**Issue by iteles** [SafeLives/SafeLives#81]
@JoshTaylor12 The call yesterday was extremely helpful, but we still need to dig into the detail here as this is the reporting will be the most complex and time-consuming single part of the system. If at all possible, you probably want to avoid building a bespoke solution here as it could become a very big piece of work!

We need a clearer view of what you will need to see in the reports that are currently generated by Snap, the data you need to manipulate and which manipulations you will need.

For example:
+ Data manipulations:
  + Selecting aggregate answers to a specific question 
  + Being able to construct complex queries to manipulate the data
  + Alphabetical sorting
+ Visuals:
  + Page breaks
  + Ability to determine which data goes into which column

**At this stage, it would be fantastically useful if one of your team was able to put together a screencast for us to understand what you need to do to generate the reports *from the point where you've merged all the data together* **.

Would that be possible?

**Comment by JoshTaylor12** [comment]
Hi Ines,

We can do this for you this afternoon. I don't think we currently have any screen capture software, is there anything you can recommend we can download for free?

**Comment by JoshTaylor12** [comment]
@iteles @harrygfox we are struggling to find anything that are systems will let us install. Would you have time this afternoon for a meeting over google hangouts in which we could share the screen and maybe you could record at your end? It will be Denise who will go through it, but if you could let me know your availability I can pass that on.

thanks

**Comment by iteles** [comment]
@JoshTaylor12 Apologies, Harry and I were in all afternoon workshops today!

I'll speak to @harrygfox first thing in the morning to check when he might be able to do this. How long do you think Denise will need?

**Comment by JoshTaylor12** [comment]
@iteles @harrygfox  not to worry! 30 minutes will be plenty of time to go through everything.

**Comment by iteles** [comment]
@JoshTaylor12 Would Denise be available at 15h00?

**Comment by JoshTaylor12** [comment]
@iteles yes 15:00 is fine. Would you be able to set up the meeting and invite Denise? her email is denise.almeida@safelives.org.uk

**Comment by JoshTaylor12** [comment]
@harrygfox are you available now?

**Comment by iteles** [comment]
Argh! I'm so sorry, this was entirely my fault. I wrote out a message to Harry asking him to set this up with Denise, but when I went back to the chat window to check how the call had gone with Harry just now, I apparently hadn't hit send 😞 

I'm very sorry about this.

I'll liaise with Denise and @harrygfox directly to get this done.
